warehouse/accounts/views.py

```diff
@@ -1514,13 +1520,17 @@ def default_response(self):
         return {
             "pending_github_publisher_form": self.pending_github_publisher_form,
             "pending_google_publisher_form": self.pending_google_publisher_form,
+            "pending_activestate_publisher_form": self.pending_activestate_publisher_form,  # noqa
             "disabled": {
                 "GitHub": self.request.flags.disallow_oidc(
                     AdminFlagValue.DISALLOW_GITHUB_OIDC
                 ),
                 "Google": self.request.flags.disallow_oidc(
                     AdminFlagValue.DISALLOW_GOOGLE_OIDC
                 ),
+                "ActiveState": self.request.flags.disallow_oidc(
+                    AdminFlagValue.DISALLOW_ACTIVESTATE_OIDC
+                ),
```

Comment:
No action required, but this is making me realize that we should probably get rid of these hardcoded provider names and replace them with something in `oidc.utils`.

Reply:
I'm going to leave this one out as there's a bit (very little) of nuance to what the change could be exactly.